eZ/Publish/Core/Repository/Repository.php

```diff
@@ -802,7 +802,6 @@ public function getSearchService()
         $this->searchService = new SearchService(
             $this,
             $this->searchHandler->contentSearchHandler(),
```

Comment:
coulød have been just "$this->searchHandler" given it only need to be one layer now right?